### ISSUE TITLE
[3.0] Godot isn't working with me - getting many error at the start of project editor

### ISSUE THREAD

**Issue by Ahmed0425** [godotengine/godot#8148]
Windows 10 - Godot 3.0
Getting this error (in attachment) at the start of the project editor
PC specifications are here (made by CPU-Z & GPU-Z)
[GPU](http://gpuz.techpowerup.com/17/03/25/4q9.png)
[DESKTOP-MHF8VDM.txt](https://github.com/godotengine/godot/files/870175/DESKTOP-MHF8VDM.txt)
[Godot 3.0.txt](https://github.com/godotengine/godot/files/870170/Godot.3.0.txt)

**Comment by eon-s** [comment]
Godot build?

Are your drivers those that comes with windows? because these use to be really bad drivers, even updated by the system (I think you need to remove the old first before installing new ones, something not easy to do on windows 10...).

I will see if I can test that on my old laptop with a radeon 6250.
On linux, (Gallium on AMD Palm), editor runs fine (maybe better than 2.x), 3D don't render materials on the testcube when running but renders fine in-editor (and that is all what I have tested so far).

**Comment by eon-s** [comment]
Surprisingly, fixnum build (March 24, 2017 at 23:38) works on windows 7 32 with a Radeon HD 6250.
Spams with invalid gl_operation after running a project but works and render things -a cube- fine, and is really slow to open and work on 3D.

Driver version is 14.502.1014 (I don't use this OS too much) from AMD site.

**Comment by Ahmed0425** [comment]
just downloaded official driver from AMD website, uninstall old one which was installed by windows update and installed the official one but didn't work :(

**Comment by vnen** [comment]
I got the same errors on Windows with a Radeon RX 480.

**Comment by Ahmed0425** [comment]
@vnen  I was using mingw compiler (didn't know differences between mingw & msvc but now I did ... lol)
I thought compiling via msvc might help so I've tried to use it now but compiling always terminated because of errors (64x Native tools were used)

**Comment by eon-s** [comment]
Tested on Windows 10 with AMD graphics, drivers 15.20.1062.1004 on integrated GPU Radeon HD 7480D.

Runs fine, the issues start when running a project, it shows the window and render 3D without problems but starts spamming "mesege" on the output and this on the console window
```
ERROR: _gl_debug_print: GL ERROR: Source: OpenGL        Type: Error     ID: 3010        Severity: High  Message: glBlitFramebuffer failed because the framebuffer configurations require that the source and destination formats and sample counts match (GL_INVALID_OPERATION)
   At: drivers/gles3/rasterizer_gles3.cpp:122
```
And keeps the spam after closing the running window.

I have seen that error with nvidia drivers too (also on linux) but not spamming them, just once or twice.

**Comment by Ahmed0425** [comment]
@eon-s How did you compile godot? MSVC or MinGW?
I was compiling with mingw and I wonder if mingw was the cause of the problem

**Comment by eon-s** [comment]
@Ahmed0425 I don't have a good setup to build for/from windows right now so I depend on fixnum builds, I think these use mingw.

If I solve some little chaos on my system I will try to make it with mingw.
Or I can test a build that fails if you have one at hand.

**Comment by eon-s** [comment]
Well, made it myself under linux with MinGW 64 and that build runs fine on both systems, same issues than fixnum's builds too.
@Ahmed0425 if you can try fixnum build and these work, then the problem could be reduced to windows compilers.

**Comment by quantum-entangler** [comment]
For me any nightly builds I've tried from different sources since early march have al crashed instantly on intel HD 4000 and give me a window filled with white and those GL_INVALID_OPERATION errors for seemingly everything when running on GeForce 750m. Windows 7 X64 SP1

**Comment by vnen** [comment]
I don't have errors at startup anymore, but with a simple scene I get an infinite amount of this;

```
ERROR: _gl_debug_print: GL ERROR: Source: OpenGL        Type: Performance
ID: 108 Severity: A
Message: glBlitFramebuffer was called with either GL_DEPTH_BUFFER_BIT or GL_STENCIL_BUFFER_BIT set while there was no valid depth/stencil attachment on the draw framebuffer
   At: drivers\gles3\rasterizer_gles3.cpp:123
```

**Comment by quantum-entangler** [comment]
Ok so I recently tried to run godot 3 from console while logging the output(from cli using >>) to a file on disk and that actually made the project selector show up correctly but the real-time logging slowed godot down to a crawl. Previously I just saw the GL ERROR's in the cli and the project selector was seemingly stuck just displaying a white window. Same on all my Windows 7 X64 machines.

**Comment by kubecz3k** [comment]
First of all thank you for your report and sorry for the delay.

We released Godot 3.0 in January 2018 after 18 months of work, fixing many old issues either directly, or by obsoleting/replacing the features they were referring to.

We still have hundreds of issues whose relevance/reproducibility needs to be checked against the [current stable version](https://godotengine.org/download), and that's where you can help us.
**Could you check if the issue that you described initially is still relevant/reproducible in Godot 3.0 or any newer version, and comment about its current status here?**

For bug reports, please also make sure that the issue contains detailed steps to reproduce the bug and, if possible, a zipped project that can be used to reproduce it right away. This greatly speeds up debugging and bugfixing tasks for our contributors.

Our Bugsquad will review this issue more in-depth in 15 days, and potentially close it if its relevance could not be confirmed.

Thanks in advance.

*Note: This message is being copy-pasted to many "stale" issues (90+ days without activity). It might happen that it is not meaningful for this specific issue or appears oblivious of the issue's context, if so please comment to notify the Bugsquad about it.*

**Comment by Megalomaniak** [comment]
For my case(RX 470) the issue has been resolved since the release candidate, however I'm not sure if the same is the case for OP's HD 6450.

Looking at gpuinfo.org I couldn't find anything in the GLES registry however openGL registry found 4 reports(in comparison): https://opengl.gpuinfo.org/gl_comparereports.php?reports_length=50&compare=compare&id%5B2132%5D=on&id%5B1685%5D=on&id%5B826%5D=on&id%5B825%5D=on

**Comment by vnen** [comment]
I also don't have this problem anymore, it was fixed before 3.0 released.

**Comment by akien-mga** [comment]
Thanks for checking.
